app/services/idv/financials_step.rb

```diff
@@ -29,6 +29,10 @@ def vendor_errors
       vendor_validator.errors if form_valid?
     end
 
+    def vendor_reasons
+      vendor_validator.reasons if form_valid?
```

Comment:
Sure. Refactored to create `vendor` namespace at the top level with key `reasons` within it. I suspect we will be adding more to the `vendor` in near future (probably `score` etc).